### ISSUE TITLE
android version port

### ISSUE THREAD

**Issue by nacho00112** [opentoonz/opentoonz#5191]
### Is your feature request related to a problem? Please describe

i want to use it in my phone.

### Describe the solution you'd like

a android port.

yeah I'm aware this is a duplicate but now that many years have passed
I'm opening this issue to make the developers reconsider this.

**Comment by RodneyBaker** [comment]
Due to the extreme difficulty in moving a program like Opentoonz to Android (and other mobile platforms) my recommendation would be to create a separate assistant application that runs on Android that can do basic tasks associated with Opentoonz and yet remains fully compatible.  This is aided by the fact that many OT files are text based XML or JSON and easily manipulated by programs on any platform.

The final product produced would still likely need to be transferred to PC, Mac or Linux but the majority of work could therefore be done on mobile applications.

Examples of processes:
- Project Management
- File Management and manipulation
- Scene Setup
- Audio Mastering
- XSheet/Timeline setup
- Image, Video and Sequence organization
- Brush Management
- Shader Design
- FX and Schematic Workflow
- Layout (spot illustrations could be designed to be automatically replaced by improved artwork)
- AI and regular assistants to aid with Pre and Post Production
- Rough drafting and outlining (basic drawing and proxy setup)
- Production Journals
- Render Management
- UI Design and Program Development
Etc. Etc.

In short, there is no lack of things that can be done on Android and other mobile platforms right now.
There is no need to wait for an android port of Opentoonz as the vast majority of things we can do in Opentoonz can be done even now on these platforms with the right tools.

What programs do you currently use on Android regularly?
What makes Android the ideal platform for your use... (what does success look like not from a program standpoint but with regard to the productions you want to produce)?
If Android cannot assist in these things now, what can be done to resolve that problem on these mobile platforms?

Opentoonz may very well have an android version in the far flung future but right now there are surely things that can be done on android to make it worth using.

**Comment by RodneyBaker** [comment]
Will transfer this report to Opentoonz-Docs where perhaps we can delve more deeply into the realm of how best to use Opentoonz in a mobile-empowered pipeline.
